thereseLYR/3D-Model-Ecommerce-Frontend
# feat: typescript conversion (clicky components)

## Changes
- typescript conversion for practice and to iron out latent bugs
- fixes an issue with cookie changes being made to the wrong cookie ID

## How to test
1) TypeScript Conversion:
    - clear all cookies
    - login and create an order, but do not check out
2) Cookie Fix
    - ensure that cookie 'temp_cart' is never created nor accessed

## Known Issues
- N/A

## Screenshots (if any)
- everything should look the same, really
